### PR TITLE
build: use direnv and automate installation of dev dependencies

### PR DESCRIPTION
**Pull request by troian**
Setup environment using [direnv](direnv.net)

1. [install direnv](https://direnv.net/docs/installation.html) if not yet using it
2. checkout this branch
3. you will be prompted to load `.envrc` with `direnv allow` command. this command has to be run only once as long as `.envrc` remains unchanged

it sets up bunch of environment variables and PATHs
1. inside akash repo `which akash` points to `<akash sources>/.cache/bin/akash` as long as it has been compiled. for example with `make akash`

2. run configurations have own `.envrc` files which making `AKASH_HOME` to point to `.cache/run/{kube|lite|single}`

- each dependency is being installed to .cache uppon it's necessity
- install k8s.io to .cache

Signed-off-by: Artur Troian <troian.ap@gmail.com>

- [x] Update README instructions for local deployment

### CODE REVIEW

File: make/setup-cache.mk [hunk 1]
```diff
@@ -7,7 +7,7 @@ $(CACHE):
 	mkdir -p $(CACHE_NODE_MODULES)
 
 $(PROTOC_VERSION_FILE): $(CACHE)
-	@echo "installing protoc compiler..."
+	@echo "installing protoc compiler v$({)PROTOC_VERSION) ..."
```
Comment (boz) [inline make/setup-cache.mk:10]: :scissors:

File: _run/common-kind.mk [hunk 2]
```diff
@@ -44,25 +44,25 @@ kind-configure-image:
     cp ./kustomize/akash-node/docker-image.yaml ./kustomize/akash-provider/docker-image.yaml
 
 .PHONY: kind-upload-image
-kind-upload-image:
-	kind --name "$(KIND_NAME)" load docker-image "${DOCKER_IMAGE}"
+kind-upload-image: $(KIND)
+	$(KIND) --name "$(KIND_NAME)" load docker-image "${DOCKER_IMAGE}"
```
Comment (boz) [inline _run/common-kind.mk:48]: I want to be able to type `kind`.  I already hate that it's now a pain in the ass to run `akash`.

Maybe we need to re-add an `env.sh` or similar.

File: go.mod [hunk 3]
```diff
@@ -45,7 +45,6 @@ require (
 	k8s.io/client-go v0.19.3
 	k8s.io/code-generator v0.19.3
 	k8s.io/metrics v0.19.3
-	sigs.k8s.io/kind v0.10.0
```
Comment (boz) [inline go.mod:48]: why do we need to put this in `.cache`?

**Review by boz — COMMENTED** [review]
What is the problem we're solving here?

File: make/init.mk [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+include $(ROOT_DIR)/.makerc
+
+# setup .cache bins first in paths to have precedence over already installed same tools for system wide use
+PATH                       := "$(PATH):$(CACHE_BIN):$(CACHE_NODE_BIN)"
+
+BUF_VERSION                ?= 0.35.1
+PROTOC_VERSION             ?= 3.13.0
+PROTOC_GEN_COSMOS_VERSION  ?= v0.3.1
+PROTOC_SWAGGER_GEN_VERSION ?= v2.2.0
```
Comment (hydrogen18) [inline make/init.mk:21]: This version doesn't produce any useful output for me (it fails entirely, actually).

See: https://github.com/ovrclk/akash/pull/1175/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R29

File: make/init.mk [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+GRPC_GATEWAY_VERSION_FILE       = $(CACHE_VERSIONS)/protoc-gen-grpc-gateway/$(GRPC_GATEWAY_VERSION)
+PROTOC_GEN_COSMOS_VERSION_FILE  = $(CACHE_VERSIONS)/protoc-gen-cosmos/$(PROTOC_GEN_COSMOS_VERSION)
+STATIK_VERSION_FILE             = $(CACHE_VERSIONS)/statik/$(STATIK_VERSION)
+MODVENDOR_VERSION_FILE          = $(CACHE_VERSIONS)/modvendor/$(MODVENDOR_VERSION)
+GIT_CHGLOG_VERSION_FILE         = $(CACHE_VERSIONS)/git-chglog/$(GIT_CHGLOG_VERSION)
+MOCKERY_VERSION_FILE            = $(CACHE_VERSIONS)/mockery/v$(MOCKERY_VERSION)
+K8S_CODE_GEN_VERSION_FILE       = $(CACHE_VERSIONS)/k8s-codegen/$(K8S_CODE_GEN_VERSION)
+
+MODVENDOR                       = $(CACHE_BIN)/modvendor
+SWAGGER_COMBINE                 = $(CACHE_NODE_BIN)/swagger-combine
```
Comment (hydrogen18) [inline make/init.mk:44]: This variable isn't used anywhere that I've found, installing it to this location doesn't do anything. I have to install nodejs, then use `npm -g` to install this package globally on my machine so the associated scripts run

Comment (troian) [reply make/init.mk:44]: it used by `script/protoc-swagger-gen.sh`

File: make/init.mk (hunk 4 above)
Comment (troian) [reply make/init.mk]: fixed

File: make/init.mk (hunk 5 above)
Comment (hydrogen18) [reply make/init.mk:44]: The line you're referring to must be this one 

```
swagger-combine ./client/docs/config.json
```

I can't find anything that actually causes the installed `swagger-combine` to be used, unless you made some change in this PR. It picks up off the users global path variable and that is it

**Review by hydrogen18 — APPROVED** [review]

**Comment by hydrogen18** [comment]
I checked out this branch and did `make setup-devenv` and it failed with this 

```
$ make setup-devenv
installing protoc compiler v3.13.0 ...
rm -f /home/ericu/tmp/akash/.cache/bin/protoc
(cd /tmp; \
curl -sOL "https://github.com/protocolbuffers/protobuf/releases/download/v3.13.0/protoc-3.13.0-linux-x86_64.zip"; \
unzip -oq protoc-3.13.0-linux-x86_64.zip -d /home/ericu/tmp/akash/.cache bin/protoc; \
unzip -oq protoc-3.13.0-linux-x86_64.zip -d /home/ericu/tmp/akash/.cache 'include/*'; \
rm -f protoc-3.13.0-linux-x86_64.zip)
rm -rf "CACHE}/versions/protoc/"
mkdir -p "CACHE}/versions/protoc/"
touch CACHE}/versions/protoc/3.13.0
Installing protoc-gen-grpc-gateway vv1.16.0 ...
rm -f /home/ericu/tmp/akash/.cache/bin/protoc-gen-grpc-gateway
curl -o "/home/ericu/tmp/akash/.cache/bin/protoc-gen-grpc-gateway" -L \
"https://github.com/grpc-ecosystem/grpc-gateway/releases/download/vv1.16.0/protoc-gen-grpc-gateway-vv1.16.0-linux-x86_64"
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100     9  100     9    0     0     27      0 --:--:-- --:--:-- --:--:--    27
chmod +x "/home/ericu/tmp/akash/.cache/bin/protoc-gen-grpc-gateway"
rm -rf "CACHE}/versions/protoc-gen-grpc-gateway/"
mkdir -p "CACHE}/versions/protoc-gen-grpc-gateway/"
touch CACHE}/versions/protoc-gen-grpc-gateway/v1.16.0
make: *** No rule to make target 'protoc-swagger', needed by 'setup-devenv'.  Stop.
```

**Comment by troian** [comment]
> I checked out this branch and did `make setup-devenv` and it failed with this
> 
> ```
> $ make setup-devenv
> installing protoc compiler v3.13.0 ...
> rm -f /home/ericu/tmp/akash/.cache/bin/protoc
> (cd /tmp; \
> curl -sOL "https://github.com/protocolbuffers/protobuf/releases/download/v3.13.0/protoc-3.13.0-linux-x86_64.zip"; \
> unzip -oq protoc-3.13.0-linux-x86_64.zip -d /home/ericu/tmp/akash/.cache bin/protoc; \
> unzip -oq protoc-3.13.0-linux-x86_64.zip -d /home/ericu/tmp/akash/.cache 'include/*'; \
> rm -f protoc-3.13.0-linux-x86_64.zip)
> rm -rf "CACHE}/versions/protoc/"
> mkdir -p "CACHE}/versions/protoc/"
> touch CACHE}/versions/protoc/3.13.0
> Installing protoc-gen-grpc-gateway vv1.16.0 ...
> rm -f /home/ericu/tmp/akash/.cache/bin/protoc-gen-grpc-gateway
> curl -o "/home/ericu/tmp/akash/.cache/bin/protoc-gen-grpc-gateway" -L \
> "https://github.com/grpc-ecosystem/grpc-gateway/releases/download/vv1.16.0/protoc-gen-grpc-gateway-vv1.16.0-linux-x86_64"
>   % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
>                                  Dload  Upload   Total   Spent    Left  Speed
> 100     9  100     9    0     0     27      0 --:--:-- --:--:-- --:--:--    27
> chmod +x "/home/ericu/tmp/akash/.cache/bin/protoc-gen-grpc-gateway"
> rm -rf "CACHE}/versions/protoc-gen-grpc-gateway/"
> mkdir -p "CACHE}/versions/protoc-gen-grpc-gateway/"
> touch CACHE}/versions/protoc-gen-grpc-gateway/v1.16.0
> make: *** No rule to make target 'protoc-swagger', needed by 'setup-devenv'.  Stop.
> ```

@hydrogen18 fixed.
make sure to run `direnv allow` after pull. there was a typo in `.env`

**Comment by hydrogen18** [comment]
I'm kinda confused at this point. I did `make setup-devenv` then `make codegen` on the latest version, it bombs on the protoc step but I'm unsure why

```
ericu@ericu-acer-laptop:~/tmp/akash$ make codegen
GO111MODULE=on go generate ./...
GO111MODULE=on go mod tidy
go mod vendor
vendoring non-go files...
/home/ericu/tmp/akash/.cache/bin/modvendor -copy="**/*.proto" -include=\
github.com/cosmos/cosmos-sdk/proto,\
github.com/cosmos/cosmos-sdk/third_party/proto
/home/ericu/tmp/akash/.cache/bin/modvendor -copy="**/*.h **/*.c" -include=\
github.com/zondax/hid
/home/ericu/tmp/akash/.cache/bin/modvendor -copy="**/swagger.yaml" -include=\
github.com/cosmos/cosmos-sdk/client/docs
./script/protocgen.sh
/home/ericu/tmp/akash/.cache/bin/protoc-gen-grpc-gateway: 1: Not: not found
--grpc-gateway_out: protoc-gen-grpc-gateway: Plugin failed with status code 127.
make: *** [make/codegen.mk:31: proto-gen] Error 1
ericu@ericu-acer-laptop:~/tmp/akash$ ls -l /home/ericu/tmp/akash/.cache/bin/protoc-gen-grpc-gateway
-rwxrwxr-x 1 ericu ericu 9 May  6 10:45 /home/ericu/tmp/akash/.cache/bin/protoc-gen-grpc-gateway

```

**Comment by troian** [comment]
@hydrogen18 fixed. don't need to run `make setup-devenv` btw. `codegen` and other targets have everything listed as dependencies

```
make clean-cache
make codegen
```

**Comment by hydrogen18** [comment]
```
$ make clean-cache
make: *** No rule to make target 'clean-cache'.  Stop.
```

There doesn't appear to be a `clean-cache` task

**Comment by troian** [comment]
sorry, it is other way round `cache-clean`

**Comment by hydrogen18** [comment]
It's still blowing up but the `swagger-combine` step is working now I think

```
direnv: loading ~/tmp/akash/.envrc
make: '/home/ericu/tmp/akash/.cache' is up to date.
direnv: export +AKASH +AKASH_DEVCACHE +AKASH_DEVCACHE_BASE +AKASH_DEVCACHE_BIN +AKASH_DEVCACHE_INCLUDE +AKASH_DEVCACHE_NODE_BIN +AKASH_DEVCACHE_NODE_MODULES +AKASH_DEVCACHE_VERSIONS +AKASH_ROOT +AKASH_RUN +GO111MODULE +GOLANG_VERSION +KIND_VERSION +ROOT_DIR ~PATH
ericu@ericu-acer-laptop:~/tmp/akash$ direnv allow
direnv: loading ~/tmp/akash/.envrc
make: '/home/ericu/tmp/akash/.cache' is up to date.
direnv: export +AKASH +AKASH_DEVCACHE +AKASH_DEVCACHE_BASE +AKASH_DEVCACHE_BIN +AKASH_DEVCACHE_INCLUDE +AKASH_DEVCACHE_NODE_BIN +AKASH_DEVCACHE_NODE_MODULES +AKASH_DEVCACHE_VERSIONS +AKASH_ROOT +AKASH_RUN +GO111MODULE +GOLANG_VERSION +KIND_VERSION +ROOT_DIR ~PATH
ericu@ericu-acer-laptop:~/tmp/akash$ make cache-clean && make codegen
rm -rf /home/ericu/tmp/akash/.cache
GO111MODULE=on go generate ./...
creating .cache dir structure...
mkdir -p /home/ericu/tmp/akash/.cache
mkdir -p /home/ericu/tmp/akash/.cache/bin
mkdir -p /home/ericu/tmp/akash/.cache/include
mkdir -p /home/ericu/tmp/akash/.cache/versions
mkdir -p /home/ericu/tmp/akash/.cache
mkdir -p /home/ericu/tmp/akash/.cache/run
installing protoc compiler v3.13.0 ...
rm -f /home/ericu/tmp/akash/.cache/bin/protoc
(cd /tmp; \
curl -sOL "https://github.com/protocolbuffers/protobuf/releases/download/v3.13.0/protoc-3.13.0-linux-x86_64.zip"; \
unzip -oq protoc-3.13.0-linux-x86_64.zip -d /home/ericu/tmp/akash/.cache bin/protoc; \
unzip -oq protoc-3.13.0-linux-x86_64.zip -d /home/ericu/tmp/akash/.cache 'include/*'; \
rm -f protoc-3.13.0-linux-x86_64.zip)
rm -rf "/home/ericu/tmp/akash/.cache/versions/protoc/"
mkdir -p "/home/ericu/tmp/akash/.cache/versions/protoc/"
touch /home/ericu/tmp/akash/.cache/versions/protoc/3.13.0
Installing protoc-gen-grpc-gateway vv1.16.0 ...
rm -f /home/ericu/tmp/akash/.cache/bin/protoc-gen-grpc-gateway
curl -o "/home/ericu/tmp/akash/.cache/bin/protoc-gen-grpc-gateway" -L \
"https://github.com/grpc-ecosystem/grpc-gateway/releases/download/v1.16.0/protoc-gen-grpc-gateway-v1.16.0-linux-x86_64"
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   650  100   650    0     0   1934      0 --:--:-- --:--:-- --:--:--  1928
100 4608k  100 4608k    0     0  3188k      0  0:00:01  0:00:01 --:--:-- 12.8M
chmod +x "/home/ericu/tmp/akash/.cache/bin/protoc-gen-grpc-gateway"
rm -rf "/home/ericu/tmp/akash/.cache/versions/protoc-gen-grpc-gateway/"
mkdir -p "/home/ericu/tmp/akash/.cache/versions/protoc-gen-grpc-gateway/"
touch /home/ericu/tmp/akash/.cache/versions/protoc-gen-grpc-gateway/v1.16.0
installing protoc-gen-cosmos v0.3.1 ...
rm -f /home/ericu/tmp/akash/.cache/bin/protoc-gen-cosmos
GOBIN=/home/ericu/tmp/akash/.cache/bin go get github.com/regen-network/cosmos-proto/protoc-gen-gocosmos@v0.3.1
rm -rf "/home/ericu/tmp/akash/.cache/versions/protoc-gen-cosmos/"
mkdir -p "/home/ericu/tmp/akash/.cache/versions/protoc-gen-cosmos/"
touch /home/ericu/tmp/akash/.cache/versions/protoc-gen-cosmos/v0.3.1
GO111MODULE=on go mod tidy
go mod vendor
installing modvendor v0.3.0 ...
rm -f /home/ericu/tmp/akash/.cache/bin/modvendor
GOBIN=/home/ericu/tmp/akash/.cache/bin GO111MODULE=on go install github.com/goware/modvendor@v0.3.0
rm -rf "/home/ericu/tmp/akash/.cache/versions/modvendor/"
mkdir -p "/home/ericu/tmp/akash/.cache/versions/modvendor/"
touch /home/ericu/tmp/akash/.cache/versions/modvendor/v0.3.0
vendoring non-go files...
/home/ericu/tmp/akash/.cache/bin/modvendor -copy="**/*.proto" -include=\
github.com/cosmos/cosmos-sdk/proto,\
github.com/cosmos/cosmos-sdk/third_party/proto
/home/ericu/tmp/akash/.cache/bin/modvendor -copy="**/*.h **/*.c" -include=\
github.com/zondax/hid
/home/ericu/tmp/akash/.cache/bin/modvendor -copy="**/swagger.yaml" -include=\
github.com/cosmos/cosmos-sdk/client/docs
./script/protocgen.sh
Installing statik v0.1.7 ...
rm -f /home/ericu/tmp/akash/.cache/bin/statik
GOBIN=/home/ericu/tmp/akash/.cache/bin GO111MODULE=on go install github.com/rakyll/statik@v0.1.7
rm -rf "/home/ericu/tmp/akash/.cache/versions/statik/"
mkdir -p "/home/ericu/tmp/akash/.cache/versions/statik/"
touch /home/ericu/tmp/akash/.cache/versions/statik/v0.1.7
installing protoc-gen-swagger v1.16.0 ...
GOBIN=/home/ericu/tmp/akash/.cache/bin GO111MODULE=on go install github.com/grpc-ecosystem/grpc-gateway/protoc-gen-swagger@v1.16.0
Installing swagger-combine...
npm install swagger-combine --prefix /home/ericu/tmp/akash/.cache
npm WARN saveError ENOENT: no such file or directory, open '/home/ericu/tmp/akash/.cache/package.json'
npm notice created a lockfile as package-lock.json. You should commit this file.
npm WARN enoent ENOENT: no such file or directory, open '/home/ericu/tmp/akash/.cache/package.json'
npm WARN .cache No description
npm WARN .cache No repository field.
npm WARN .cache No README data
npm WARN .cache No license field.

+ swagger-combine@1.3.0
added 24 packages from 24 contributors and audited 24 packages in 2.787s
found 0 vulnerabilities

./script/protoc-swagger-gen.sh
/home/ericu/tmp/akash/.cache/bin/statik -src=client/docs/swagger-ui -dest=client/docs -f -m
Swagger docs are in sync
installing k8s code-generator v0.19.3 ...
rm -f /home/ericu/tmp/akash/.cache/bin/go-to-protobuf
GOBIN=/home/ericu/tmp/akash/.cache/bin go install /home/ericu/tmp/akash/vendor/k8s.io/code-generator/...
rm -rf "/home/ericu/tmp/akash/.cache/versions/k8s-codegen/"
mkdir -p "/home/ericu/tmp/akash/.cache/versions/k8s-codegen/"
touch /home/ericu/tmp/akash/.cache/versions/k8s-codegen/v0.19.3
chmod +x /home/ericu/tmp/akash/vendor/k8s.io/code-generator/generate-groups.sh
GOBIN=/home/ericu/tmp/akash/.cache/bin /home/ericu/tmp/akash/vendor/k8s.io/code-generator/generate-groups.sh all \
github.com/ovrclk/akash/pkg/client github.com/ovrclk/akash/pkg/apis \
akash.network:v1
go: downloading golang.org/x/tools v0.0.0-20200616133436-c1934b75d054
go: downloading golang.org/x/xerrors v0.0.0-20191204190536-9bdfabe68543
Generating deepcopy funcs
F0506 14:54:35.592629  307506 deepcopy.go:131] Failed loading boilerplate: open k8s.io/code-generator/hack/boilerplate.go.txt: no such file or directory
goroutine 1 [running]:
k8s.io/klog/v2.stacks(0xc0000bc001, 0xc011def2c0, 0x99, 0xe9)
	/home/ericu/go/pkg/mod/k8s.io/klog/v2@v2.2.0/klog.go:996 +0xb9
k8s.io/klog/v2.(*loggingT).output(0x964660, 0xc000000003, 0x0, 0x0, 0xc0128db180, 0x8292db, 0xb, 0x83, 0x0)
	/home/ericu/go/pkg/mod/k8s.io/klog/v2@v2.2.0/klog.go:945 +0x191
k8s.io/klog/v2.(*loggingT).printf(0x964660, 0x3, 0x0, 0x0, 0x76a406, 0x1e, 0xc0140ffae8, 0x1, 0x1)
	/home/ericu/go/pkg/mod/k8s.io/klog/v2@v2.2.0/klog.go:733 +0x17a
k8s.io/klog/v2.Fatalf(...)
	/home/ericu/go/pkg/mod/k8s.io/klog/v2@v2.2.0/klog.go:1456
k8s.io/gengo/examples/deepcopy-gen/generators.Packages(0xc012756460, 0xc0000c6fa0, 0x747a9b, 0x6, 0xc012756460)
	/home/ericu/go/pkg/mod/k8s.io/gengo@v0.0.0-20200428234225-8167cfdcfc14/examples/deepcopy-gen/generators/deepcopy.go:131 +0xfd
k8s.io/gengo/args.(*GeneratorArgs).Execute(0xc0000c6fa0, 0xc0140ffe38, 0x747a9b, 0x6, 0x775000, 0x0, 0x0)
	/home/ericu/go/pkg/mod/k8s.io/gengo@v0.0.0-20200428234225-8167cfdcfc14/args/args.go:206 +0x1b7
main.main()
	/home/ericu/tmp/akash/vendor/k8s.io/code-generator/cmd/deepcopy-gen/main.go:77 +0x46b

goroutine 19 [chan receive]:
k8s.io/klog/v2.(*loggingT).flushDaemon(0x964660)
	/home/ericu/go/pkg/mod/k8s.io/klog/v2@v2.2.0/klog.go:1131 +0x8b
created by k8s.io/klog/v2.init.0
	/home/ericu/go/pkg/mod/k8s.io/klog/v2@v2.2.0/klog.go:416 +0xd8
make: *** [make/codegen.mk:25: kubetypes] Error 255
****

``` 
I tried running `make modvendor` but it didn't change anything

File: Makefile [hunk 6]
```diff
@@ -1,7 +1,6 @@
-BINS                  := akash akash_docgen
```
Comment (boz) [inline Makefile:1]: :pray:

File: _run/common-commands.mk [hunk 7]
```diff
@@ -17,109 +17,109 @@ LEASE_SERVICES ?= web
 
 .PHONY: multisig-send
 multisig-send:
-	$(AKASHCTL) tx send \
-		"$(shell $(AKASHCTL) $(KEY_OPTS) keys show "$(MULTISIG_KEY)" -a)" \
-		"$(shell $(AKASHCTL) $(KEY_OPTS) keys show "$(KEY_NAME)"     -a)" \
+	$(AKASH) tx send \
```
Comment (boz) [inline _run/common-commands.mk:20]: :pray:

File: x/deployment/client/cli/tx.go [hunk 8]
```diff
@@ -319,7 +319,7 @@ func cmdGroupStart(_ string) *cobra.Command {
 	cmd := &cobra.Command{
 		Use:     "start",
 		Short:   "start a Deployment's specific Group",
-		Example: "akashctl tx deployment group pause --owner=[Account Address] --dseq=[uint64] --gseq=[uint32]",
+		Example: "akash tx deployment group pause --owner=[Account Address] --dseq=[uint64] --gseq=[uint32]",
```
Comment (boz) [inline x/deployment/client/cli/tx.go:322]: :pray:

**Review by boz — CHANGES_REQUESTED** [review]
Looks great!

1. `$(AKASH)` does not seem to be defined for me, so `make init` is always failing [here](https://github.com/ovrclk/akash/blob/6c8b5c41deb3054fe9943ee28ca493812ee85e40/_run/common.mk#L49-L51)
2. Can we make `make bins` be the default for the top-level `make`?

**Comment by troian** [comment]
1. fixed
2. 💯 . made it

File: .github/workflows/k8s-integration.yaml [hunk 9]
```diff
@@ -31,9 +34,17 @@ jobs:
       - name: Docker Status
         run: docker ps -a
       - name: Setup Ingress K8S
-        run: make -s -C _run/kube kind-ingress-setup
+        run: |-
+          trap "cd ../.." EXIT
```
[the remark this answers is not in the document]
Comment (troian) [reply .github/workflows/k8s-integration.yaml]: cleaned

**Review by boz — APPROVED** [review]
:rocket:

**Pull request merged**